lightning/src/ln/channel.rs

```diff
@@ -803,9 +807,23 @@ macro_rules! secp_check {
 }
 
 impl<Signer: Sign> Channel<Signer> {
-	// Convert constants + channel value to limits:
-	fn get_holder_max_htlc_value_in_flight_msat(channel_value_satoshis: u64) -> u64 {
-		channel_value_satoshis * 1000 / 10 //TODO
+	/// Returns the value to use for `holder_max_htlc_value_in_flight_msat` as a percentage of the
+	/// `channel_value_satoshis` in msat, based on how many of the percent its been configured to
+	/// be set to by the user.
```

Comment:
s/based on how many of the percent its been configured to be set to by the user/[`ChannelHandkshakeConfig::max_inbound_htlc_value_in_flight_percent_of_channel`] (double check that link lol)

Reply:
Thanks, edited it to:
set through [`ChannelHandkshakeConfig::max_inbound_htlc_value_in_flight_percent_of_channel`] 
But can remove the "set through" if you prefer :)!